### PR TITLE
ensure that pineappl and lhapdf exist before installing vrap

### LINKED ISSUE #147: Check for pineappl before installing vrap
At the moment this is not the case. We want to automatize this right? 

https://github.com/NNPDF/runcards/blob/e3601ef290d9eef2e21af9454f8e5a88f900e2e8/runcardsrunner/cli/install.py#L32

### PR DESCRIPTION
**Pull request by scarlehoff**
Closes #147

### CODE REVIEW

**Comment by AleCandido** [comment]
The function to install LHAPDF I believe is still there:
https://github.com/NNPDF/runcards/blob/91614524f1efdf171a01243749bd23de1daf9638/pinefarm/install.py#L281-L327

So scratch the [package dependency](https://github.com/NNPDF/runcards/blob/91614524f1efdf171a01243749bd23de1daf9638/pyproject.toml#L35) and let's go for the usual one.

Unfortunately, the package was already falling short, because it doesn't set up the data location (part of install procedure, done with `make install`), and was not even exposing the CLI (thus I was using `lhapdf_management`, that is temporarily dead).

I'd like to have a new package to do all these stuffs, but I do not decide on my own (nor at the moment is a priority/I have time to invest on it). Let's scratch packages for LHAPDF (both, actually) and let's go for the officially supported way through autotools (a bit more painful and less integrated, but it was already working before...).

**Comment by scarlehoff** [comment]
ok, i'll remove the lhapdf python packages then

**Comment by scarlehoff** [comment]
Mm, not that easy. `lhapdf` is a top level import for various things so solving that require redesigning a bit many pieces.

**Comment by scarlehoff** [comment]
I'm happy like this. I guess the user might get a second installation of lhapdf if they want to use vrap but that's just a fact of life.

File: pinefarm/install.py [hunk 1]
```diff
@@ -181,9 +185,8 @@ def cli_installed():
         return True
 
     print("Installing...")
-
-    if not pkgconfig.exists("lhapdf"):
```
Comment (AleCandido) [inline pinefarm/install.py:185]: Why did you remove the `pkgconfig` check?

Comment (scarlehoff) [reply pinefarm/install.py:185]: It's doing it twice.

**Comment by AleCandido** [comment]
Ok, the PR is good enough for me, if it solves the problem we can also merge it.

The two things that remain:
1. LHAPDF packages never fully worked, we should go back to a regular installation (but yes, possibly we need to scope some imports); this might still be breaking stuffs, since LHAPDF might be detected but "the wrong one"
2. I'm not fully happy about the "dependencies": they are a bit hidden inside the functions; nothing we need to deal now, but maybe I'd implement a very basic "dependency manager": a dictionary (containing mutual dependencies), and when start installing things we populate a list, and iterate over it to install stuffs sorted (it should be simple enough, but more explicit)

**Review by AleCandido — APPROVED** [review]

**Comment by scarlehoff** [comment]
> a dictionary (containing mutual dependencies), and when start installing things we populate a list, and iterate over it to install stuffs sorted (it should be simple enough, but more explicit)

We can use some catchy name for it. Mmm.
Something like... ValidPine? :innocent:

**Comment by AleCandido** [comment]
> We can use some catchy name for it. 
> Mmm. Something like... ValidPine? :innocent:

I said "simple" and "explicit" :)
